src/main/scala/io/gatling/sbt/settings/gatling/TaskEnterpriseStart.scala

```diff
@@ -16,40 +16,60 @@
 
 package io.gatling.sbt.settings.gatling
 
-import scala.util.{ Failure, Success, Try }
+import scala.util.*
 
 import io.gatling.plugin.EnterprisePlugin
 import io.gatling.plugin.model.RunSummary
-import io.gatling.sbt.GatlingKeys._
-import io.gatling.sbt.settings.gatling.EnterprisePluginTask._
-import io.gatling.sbt.settings.gatling.EnterpriseUtils._
+import io.gatling.sbt.GatlingKeys.*
+import io.gatling.sbt.settings.gatling.EnterpriseUtils.InitializeInputTask
+import io.gatling.sbt.settings.gatling.TaskEnterpriseStart.CommandArgs.CommandArgsParser
 
 import sbt.{ Configuration, Def }
-import sbt.Keys._
-import sbt.complete.DefaultParsers._
+import sbt.Keys.*
+import sbt.complete.DefaultParsers.*
```

Comment:
All the `_` are switched to `*`. I know it's an automatic decision of your IDE, but are we sure to use it, or it's a mistake?